sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ChangeFeedProcessorTest.java

```diff
@@ -81,11 +81,11 @@ public void readFeedDocumentsStartFromBeginning() {
             ChangeFeedProcessorTest.log.info("END processing from thread {}", Thread.currentThread().getId());
         }; 
         changeFeedProcessor = ChangeFeedProcessor.changeFeedProcessorBuilder()
-            .setHostName(hostName)
-            .setHandleChanges(itemConsumer)
-            .setFeedContainer(createdFeedCollection)
-            .setLeaseContainer(createdLeaseCollection)
-            .setOptions(new ChangeFeedProcessorOptions()
+            .hostName(hostName)
+            .handleChanges(itemConsumer)
+            .feedContainer(createdFeedCollection)
+            .leaseContainer(createdLeaseCollection)
+            .options(new ChangeFeedProcessorOptions()
                 .setLeaseRenewInterval(Duration.ofSeconds(20))
```

Comment:
it looks like the ChangeFeedProcessorOptions still has setters. Should we rename those as well to be consistent with the other changes? BTW my preference is to prefix these using "with" but that's probably for a later SDK version :-)

Reply:
I feel we are going back and forth with removing `set` as prefix and then adding it back again and then removing it again. !!

Reply:
The resulting ChangeFeedProcessor object is immutable, it cannot be updated after it was built. ChangeFeedProcessorOptions might look like a POJO but in this case it acts almost as a builder; the getters are only used by the internal implementation.

Let's stick with what we have, and get this SDK GA'ed as soon as we can.

And for those curious, some background story about my comment. In the ARM Java Fluent SDK we used with* for methods returning "this", and could be chained in a Fluent/builder like pattern. We also accept setters and getters, the difference was that the setters will have to return "void" and never "this", while for the getters the explicit contract was that a network/long operation call must be made in order to return the value.

Reply:
Sounds good Milis. Thanks for the explanation, appreciate it.